pkgs/development/python-modules/fugashi/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, buildPythonPackage
+, cython
+, mecab
+, setuptools-scm
+, ipadic
+, unidic
+, unidic-lite
+}:
+
+buildPythonPackage rec {
+  pname = "fugashi";
+  version = "1.2.1";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "polm";
+    repo = "fugashi";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-VDqRhJiNDbKFE284EAUS0d5T9cl8kgyHjh+r/HjjDY8=";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [ cython mecab setuptools-scm ];
```

Comment:
does the library compile against mecab? Then it needs to go to buildInputs

Reply:
The library is a cython wrapper around mecab, it doesn't seem to be used at runtime.
I've tried moving mecab to buildInputs but the resulting builds fail.